app/services/rover_command_runner.rb

```
@@ -26,7 +26,8 @@ def call(instructions)
         @rover.turn_right
       when 'M'
         @rover.move_forward
+      when 'S'
```

RuboCop (1.73)

[warning] Avoid `when` branches without a body.

(Lint/EmptyWhen)